src/main/java/uk/gov/homeoffice/digital/sas/timecard/validators/timeentry/TimeEntryValidator.java

```diff
@@ -68,9 +68,9 @@ private List<TimeEntry> getClashingTimeEntries(TimeEntry timeEntry) {
     session.setHibernateFlushMode(FlushMode.MANUAL);
 
     var timeEntryClashes = BeanUtil.getBean(TimeEntryRepository.class).findAllClashingTimeEntries(
-        timeEntry.getOwnerId() != null ? timeEntry.getOwnerId().toString() : null,
-        timeEntry.getId() != null ? timeEntry.getId().toString() : null,
-        timeEntry.getTenantId() != null ? timeEntry.getTenantId().toString() : null,
+        timeEntry.getOwnerId() != null ? timeEntry.getOwnerId() : null,
+        timeEntry.getId() != null ? timeEntry.getId() : null,
+        timeEntry.getTenantId() != null ? timeEntry.getTenantId() : null,
```

Comment:
nit: I don't think we need the null check anymore as we no longer need to do the string conversion.